readme.md

```diff
@@ -43,6 +43,14 @@ For example, if you are working on Spanish, and started the work on July 13th, 2
 
 Once your PR is accepted and merged into "main", the branch is removed from the repo. Create a new branch for next batch of changes.
 
+## Committing Changes
+Before committing your changes to the repo, you need to run the following commands: 
+
+`npm run generate:app` to auto generate required code
```

Comment:
Contributors can run `npm run generate` which includes `app` and `notifications`.